README.md

```diff
@@ -396,14 +397,59 @@ authClient.signIn({
 
 ### `signOut()`
 
-Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example).
+Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example) and clears all tokens stored in the `TokenManager`.
+
+By default:
+
+  - the user will be signed out using an API method 
+  - the page location will not be changed. 
+  
+When calling `signOut` with the default options, it is generally recommended to reload the page after the promise resolves to ensure your app is properly re-initialized in an unauthenticated state.
+
+Since the API method relies upon an XHR call and the ability to set 3rd-party cookies, it may fail in certain scenarios, leaving the user's remote session open. For this reason, we generally recommend using the redirect method over the API method. The redirect method sets the window location to Okta where the session and associated cookies will be fully cleared.
```

Comment:
I heard something about incognito mode in Safari, I haven't investigated the 3rd party issue, but that's the main one I'm aware of

Reply:
We've definitely had reports of failure in Chrome with the "block 3rd party cookies" setting. 

Incognito (privacy) mode Safari sets storage quota to 0 for localStorage/sessionStorage, so that would fail if we need something from that storage, but I've not seen reports of it.

As blocking 3rd party cookies gets more popular (if it does), it will cause more problems when calling the API that relies on a cookie, as opposed to redirecting.